sites/gc-page-feedback/_base.scss

```diff
@@ -0,0 +1,31 @@
+/*
+  @title: Page feedback form - Base
+ */
+
+// In noscript/basic HTML mode...
+.no-js,
+.wb-disable {
+	.gc-pg-hlpfl {
+		// Left-align the legend and yes/no buttons
+		.text-left-nojs {
+			text-align: left;
+		}
+
+		// Disable row gutters (default selector isn't specific-enough)
+		.row-no-gutters {
+			[class*="col-"] {
+				padding-left: 0;
+				padding-right: 0;
+			}
+		}
```

Comment:
Is there a way to achieve this without adding this piece of code? Maybe there isn't. @duboisp thoughts?

Reply:
Unfortunately I can't think of a better way than this :(.

The problem is that the widget's noscript/basic HTML grid selectors are more specific than [Bootstrap 3.4's ``row-no-gutters`` selectors](https://github.com/twbs/bootstrap-sass/blob/b34765d8a6aa775816c59012b2d6b30c4c66a8e9/assets/stylesheets/bootstrap/_grid.scss#L43-L51).

The need for this selector is a byproduct of the ``col-sm-12-nojs`` and ``col-md-12-nojs`` classes. Those classes are declared inside of the ``.no-js, .gc-pg-hlpfl`` / ``.wb-disable .gc-pg-hlpfl`` selectors and extend Bootstrap 3.4's vanilla versions of ``col-sm-12`` and ``col-md-12``.

For example, when my ``.col-md-12-nojs`` class extends ``.col-md-12``, it causes 15px left/right padding to be borrowed from Bootstrap's grid selector. So far, so good. But... that puts the borrowed properties into my selector - which is more specific than ``.row-no-gutters [class*="col-"]``. So my extend overtakes Bootstrap's override.

Reply:
It might be worthwhile to try more SASS extend tests just to be sure.

My memory of that is pretty vague, but I don't think I would've skipped using an extend unless something forced me to.

Reply:
Looks like ``@extend`` works fine in this scenario, so I've switched over to that.

The reason I manually-redeclared part of the selector originally was probably because I wanted to reduce the amount of redundant property declarations in generated CSS.

``@extend`` is simpler to write and more modular, but has the downside of causing the [main ``.row-no-gutters`` class' non-conflicting properties](https://github.com/twbs/bootstrap-sass/blob/b34765d8a6aa775816c59012b2d6b30c4c66a8e9/assets/stylesheets/bootstrap/_grid.scss#L43-L45) to get re-declared using more specific-selectors in generated CSS. So it's kind of like a pick-your-poison scenario :(.